paparazzi/src/test/java/app/cash/paparazzi/HtmlReportWriterTest.kt

```diff
@@ -385,6 +385,71 @@ class HtmlReportWriterTest {
     }
   }
 
+  @Test
```

Comment:
doesn't this test already exist above from the previous PR that was merged in?

Reply:
That test verified that the snapshots are saved correctly, this one verifies that the files in the report folder are also correct. This fixes the bug where recording snapshots with the same hash would display the same image in the paparazzi html report because they generated the same file name